products/proxmox-ve.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Proxmox VE
+category: os
+iconSlug: proxmox
+permalink: /proxmox-ve
+alternate_urls:
+-   /pve
+-   /proxmoxve
+activeSupportColumn: Supported
+releasePolicyLink: https://pve.proxmox.com/wiki/Roadmap
+releaseDateColumn: true
+eolColumn: true
```

Comment:
Can be removed. eolColumn must be set only if you want to override de column label (which is `Security Support` by default).

With this value the label is `true`. It can be seen on the preview deployed for this PR: https://deploy-preview-2140--endoflife-date.netlify.app/proxmox-ve.